### ISSUE TITLE
copay details screen A11y issues - High - All - Voiceover / Talkback not speaking the correct information

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#12107]
## What happened?

Some A11Y announcements with voiceover/talkback aren't being announced correctly on the copay details screen.

See screen shots below that have specifics called out.

## Specs:

- Device: 
- OS: iOS / Android
- User Account: +81
- Accessibility State: Voiceover / talkback

## Steps to Reproduce

Visit the items with voiceover / talkback turned on.
See screen shots detailing issues.

## Desired behavior

1. Collapsed tab should announce the state upon first visit:  Collapsed.

2. Current statement can only apply to one of the statement links, and only if it is current.

## Acceptance Criteria

See the PR ticket:  https://github.com/department-of-veterans-affairs/va-mobile-app/pull/11809

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact:  Low
- Frequency: High 

- 2 - High

## Screen shot(s) and additional information

<img width="724" height="788" alt="Image" src="https://github.com/user-attachments/assets/f8a6e630-5aad-419b-b3b4-679798f010f1" />

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by kimmccaskill-oddball** [comment]
Statement voiceover will be fixed along with rest of copay contents in 
https://github.com/department-of-veterans-affairs/va-mobile-feature-support/issues/939
